datasets/us/sec_harmed_investors/us_sec_harmed_investors.yaml

```diff
@@ -0,0 +1,698 @@
+title: US SEC actions due to harmed Investors
+entry_point: crawler.py
+prefix: us-sec-harm
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Enforcement actions against persons and firms that had harmed investors
+description: |
+  This dataset contains information about SEC enforcement actions where
+  the defendants or respondents were ordered to pay financial judgments.
+  In these instances, the SEC may work to distribute these funds to investors
+  who were harmed.
+publisher:
+  name: U.S. Securities and Exchange Commission
+  description: Mexico's Senate
```

Comment:
I copied from [here](https://github.com/opensanctions/opensanctions/blob/d74f6e6bcd98ef441a3771632f52f78f244088bc/datasets/us/sec_pause/us_sec_pause.yml#L30). I usually use previous crawlers as a template, I think it's easier just to change the fields rather than rewriting the YAML from scratch. But this time I forgot to edit this field, sorry!

Reply:
I do the same :)